config/install/README.md

```diff
@@ -0,0 +1,146 @@
+# Install Kaudrant and Sail via OLM
+
+- Pre-Req is that OLM (operator lifecycle manager) is already installed
+
+- (optional dependencies)
+  - If you want to use `TLSPolicy` you should install the cert-manager operator. 
+  - AWS/Azure or GCP with DNS capabilities if you want to make use of `DNSPolicy`.
+  - Accessible Redis instance, if you want persistent storage for your rate limit counters.
+
+
+Install the Sail and Kuadrant Operators via OLM:
+
+
+> Note: By default this will install the "latest" or "main" of kuadrant. To change that, pick a release from the releases page in the kuadrant operator and change the image in the `config/deploy/install/standard/kustomization.yaml`.
+
+> Note: We are using the `--context` flag here this is useful if installing on more than one cluster otherwise it is not needed
+
+```
+export ctx=replace-with-your-context
+```
+
+```
+kubectl apply -k config/install/standard --context=$ctx
+``` 
+
+3) verify kuadrant and sail operators are installed. Note this can take a while. You can also take a look at the subscription and installplan resource to help with debugging but the end state should be as below:
+
+```
+kubectl get deployments -n kuadrant-system --context=$ctx
+```
+
+```
+
+NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
+authorino-operator                      1/1     1            1           83m
+dns-operator-controller-manager         1/1     1            1           83m
+kuadrant-console-plugin                 1/1     1            1           83m
+kuadrant-operator-controller-manager    1/1     1            1           83m
+limitador-operator-controller-manager   1/1     1            1           83m
+
```

Comment:
```suggestion
```suggestion
kubectl get deployments -n kuadrant-system --context=$ctx

# NAME                                    READY   UP-TO-DATE   AVAILABLE   AGE
# authorino-operator                      1/1     1            1           83m
# dns-operator-controller-manager         1/1     1            1           83m
# kuadrant-console-plugin                 1/1     1            1           83m
# kuadrant-operator-controller-manager    1/1     1            1           83m
# limitador-operator-controller-manager   1/1     1            1           83m
```
Looks cleaner than the code block like I originally thought as the copy button makes it look like its something to be run

Reply:
markdown in the suggestion option is a little odd as its markdown suggestions in markdown so you may need to make the changes manually